projects/visual_recognition_grid_cells/python2_htm_docker/docker_dir/train_and_eval_GridCellNet.py

```diff
@@ -73,21 +73,21 @@
 
 
 def object_learning_and_inference(
-    EVAL_ON_TRAINING_DATA_BOOL,  # noqa: N803
-    locationModuleWidth,
-    feature_columns_to_grid_cells_threshold,
-    grid_cells_to_feature_columns_threshold,
-    bumpType,
-    cellCoordinateOffsets,
-    cellsPerColumn,
-    activeColumnCount,
-    columnCount,
-    featuresPerObject,
-    objectWidth,
-    numModules,
-    seed1,
-    seed2,
-    anchoringMethod):
+        EVAL_ON_TRAINING_DATA_BOOL,  # noqa: N803
+        locationModuleWidth,
+        feature_columns_to_grid_cells_threshold,
```

Comment:
If the `inferObjectWithRandomMovements` had noqa: N803 for all camel-cased attributes, shouldn't it be the same here? Or have it on a file level?

Reply:
I'm not really sure how this file passed `flake8` and code review.
IMHO, we should not mix snake-case and camel-case.